api/envoy/http/backend_routing/config.proto

```diff
@@ -19,26 +19,30 @@ package google.api.envoy.http.backend_routing;
 import "validate/validate.proto";
 
 message BackendRoutingRule {
+  reserved 2;
```

Comment:
Are we going to make this a breaking change? if so,  it is fine.
we need to have a doc to list all deprecated fields.

If we need support backward compatible,  we may have to keep it and support it.

we also need a doc to list all fields that waiting to be deprecated.  so next break change can deprecate all of them.

Reply:
Yes this will be a breaking change. With API Config Versioning, we don't have to support both. 

What sort of doc are you thinking of having? I don't think API Gateway needs it, as they will use the correct version of config generator to generate the new config for the new envoy.

Reply:
Maybe just config_breaking_change.md in the top folder to list out the config version and which fields causing the breakage.

Reply:
Ack